src/command/controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Command } from 'commander';
+import { create, findAll } from '../service/service';
+import ListUser from '../model/request/list.user.type';
+import CreateUser from '../model/request/create.user.type';
+import { validate, validateWithMax } from '../utils/validator';
+import { modules } from '../dependencies';
+
+export const program = new Command();
+
+program
+  .name('lovely-stay-cli')
+  .description('lovely stay challenge')
+  .version('1.0.0');
+
+program.command('create')
+  .description('Create an user based on data from github')
+  .requiredOption('-u, --username <string>', 'github username')
```

Comment:
What is a problem that could happen with these two next lines? How should promises be handled?

Reply:
Nice catch! I actually forgot to update this portion of the solution. Commander supports ```program.parseAsync(process.argv)``` and then we can make those command functions also async